recipes/recipe-lemongrass-portobello-steak.html

```diff
@@ -0,0 +1,170 @@
+                    <li><a href="../resources.html">Resources</a></li>
+                    <li><a href="../contact.html">Contact</a></li>
+                </ul>
+            </nav>
+
+            
+            <div class="featured_recipe" id="recipe_page">
+                <div class="title_text">
+                    <a href="../recipes.html#recipes"><i class="fas fa-chevron-left"></i></a>
+                    <h2>Lemongrass Portobello Mushroom Steak</h2>
+                </div>
+            </div>
+                
+            <div class="recipe_image_text_section">
+                <img class="photo left" src="../images/recipes/Lemongrass-portobello-steak.jpeg" alt="Recipe"/>
+<!--                <iframe class="recipe-video left" src="https://www.youtube.com/embed/hSJuW2qgmfk" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>-->
+                <div class="recipe_body right">
+                    <div>
+                        <h3>Recipe submitted by Ann</h3>
+                        <p class="recipe_info">
+                            <i class="fas fa-seedling"></i>Vegan&ensp;   
+    <!--                        <i class="fas fa-carrot"></i>Vegetarian&ensp;-->
+                            <i class="fas fa-ban"></i>Gluten-Free&ensp;
+                            <i class="fas fa-circle"></i>Easy&ensp;
+    <!--                        <i class="fas fa-square"></i>Moderate&ensp;-->
+    <!--                        <i class="fas fa-star"></i>Difficult&ensp;-->
+                            <br>
+                            <i class="fas fa-clock"></i>30 mins&ensp;
+                            <i class="fas fa-user"></i>3-4 Servings
+                        </p>
+<!--
+                        <p>Description: 
+                            <a href="https://www.youtube.com/watch?v=tUi4_ektVn0" target="_blank"> Video 1</a>
+                        </p>
+-->
+                    </div>
+                </div>
+            </div>
+            
+            
+            <div class="recipe_image_text_section" id="recipe_image_text_section_body">
+                
+                <div class="recipe_text left">
+                    <h3>Ingredients</h3>
+                    <ul>
+                        <li>4 large portobello mushrooms</li>
+                        <li>1-2 Stalk lemongrass</li>
+                        <li>2-3 Kaffir lime leaves</li>
+                        <li>2-3 Cilantro stems</li>
+                        <li>1 tbsp coconut amino sauce</li>
+                        <li>1 tsp salt</li>
+                        <li>1/2 tsp chilli powder</li>
+                        <li>1 tsp black pepper</li>
+                        <li>1-2 medium sweet potatoes</li>
+                        <li>1 broccoli head</li>
+                        <li>1 tbsp nutritional yeast (optional) </li>
+            
+                    </ul>
```

Comment:
I think the way you grouped the ingredients and wrote the directions makes more sense, good job!